client/my-sites/all-sites-icon/index.jsx

```diff
@@ -31,13 +31,13 @@ export default React.createClass( {
 	getIcons() {
 		let sites = union( this.getSitesWithIcons(), this.getMaxSites() ).slice( 0, MAX_ICONS );
 		return sites.map( function( site ) {
-			return <SiteIcon site={ site } key={ site.ID + '-icon' } size={ 25 } />;
+			return <SiteIcon site={ site } key={ site.ID + '-icon' } size={ 16 } />;
 		} );
 	},
 
 	render() {
 		const icons = this.getIcons();
-		const classes = 'all-sites-icon is-shape-' + this.getMaxSites().length;
+		const classes = `all-sites-icon has-${this.getMaxSites().length}-icons`;
```

Comment:
Do [`is-shape-` modifier classes](https://github.com/Automattic/wp-calypso/blob/82ddc3b336901815202101b8e3c506d66883bdfb/client/my-sites/all-sites-icon/style.scss#L17-L37) need to be updated/removed? In the case we're not applying styles, should we just omit the modifier class?